pkg/backup/backup.go

```diff
@@ -298,23 +332,3 @@ func getMemberWithMaxRev(pods []*v1.Pod, tc *tls.Config, selfHosted bool) (*etcd
 	}
 	return member, maxRev
 }
-
-func (b *Backup) getLatestBackupRev() int64 {
```

Comment:
do not move this around. just change the receiver to backupmanager to ensure minimal change.